### PR TITLE
PVP-182 and PVP-175

### PR DESCRIPTION
**Pull request by KhajiitLikeToSneak**
Added:
- Show text that indicates there are no tasks of selected type
- Show task creation form upon clicking a day that has no tasks

### CODE REVIEW

File: app/src/main/java/com/pvp/app/ui/screen/calendar/CalendarMonthlyScreen.kt [hunk 1]
```diff
@@ -53,11 +55,20 @@ fun CalendarMonthlyScreen(
...
 
     if (showDialog) {
-        DayDialog(
-            date = selectedDate,
-            tasks = dateTasks,
-            onDismissRequest = { showDialog = false }
-        )
+        if (dateTasks.isEmpty()) {
+            CreateTaskDialog(
+                date = LocalDateTime.of(selectedDate, LocalTime.now()),
```
Comment (klaidoshka) [inline app/src/main/java/com/pvp/app/ui/screen/calendar/CalendarMonthlyScreen.kt:60]: Split params.

File: app/src/main/java/com/pvp/app/ui/screen/calendar/CalendarWeeklyScreen.kt [hunk 2]
```diff
@@ -254,23 +261,43 @@ fun Day(
...
 
         if (expand) {
-            Spacer(modifier = Modifier.padding(8.dp))
-
-            TaskFilterBar(selectedFilter) { filter ->
-                selectedFilter = filter
-            }
+            if (!tasks.any()) {
+                CreateTaskDialog(
+                    date = LocalDateTime.of(date, LocalTime.now()),
```
Comment (klaidoshka) [inline app/src/main/java/com/pvp/app/ui/screen/calendar/CalendarWeeklyScreen.kt:266]: Split params.

File: app/src/main/java/com/pvp/app/ui/screen/calendar/CalendarWeeklyScreen.kt [hunk 3]
```diff
@@ -254,23 +261,43 @@ fun Day(
...
+                LazyColumn(
+                    modifier = Modifier.width(screenWidth)
+                ) {
+                    if (!filteredTasks.any()) {
+                        item {
+                            Text(
+                                fontStyle = FontStyle.Italic,
+                                modifier = Modifier.padding(8.dp),
+                                style = MaterialTheme.typography.bodyMedium,
+                                text = "No ${selectedFilter.toString().toLowerCase()} tasks for this day"
```
Comment (klaidoshka) [inline app/src/main/java/com/pvp/app/ui/screen/calendar/CalendarWeeklyScreen.kt:291]: `No <type> tasks setup for this day` may be more correct?

**Review by klaidoshka — APPROVED** [review]

**Review by vytjar — APPROVED** [review]
No task text should be horizontally centered.

**Pull request merged**